README.md

```diff
@@ -11,6 +11,13 @@ cd app
 yarn # to install dependencies
 yarn dev # run the application
 ```
+## Docker
+To run application using Docker, run the following commands from the root folder:
+```bash
+docker build . -f app/Dockerfile # will return 'Successfully built <IMAGE>'
```

Comment:
While it's an anti-pattern to use the `latest` tag when we're deploying or pushing images around, it can be helpful to use this tag locally since on a development machine we always know when and how the last build was triggered.
```suggestion
docker build . -f app/Dockerfile connectivity-intake:latest
```